src/services/bunker_cases/abnormal_cl_rebase.py

```diff
@@ -85,7 +85,10 @@ def _calculate_lido_normal_cl_rebase(self, blockstamp: ReferenceBlockStamp) -> G
         )
 
         normal_cl_rebase = AbnormalClRebase.calculate_normal_cl_rebase(
-            self.b_conf, mean_all_effective_balance, mean_lido_effective_balance, epochs_passed_since_last_report
+            self.b_conf,
+            max(EFFECTIVE_BALANCE_INCREMENT, mean_all_effective_balance),  # to avoid division by zero
```

Comment:
not sure it belongs here.
Zero should be handled by calculate_normal_cl_rebase correctly

Reply:
To be honest, I didn't see any difference

Reply:
Maybe handle zero inside?

Reply:
done